kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -316,11 +316,16 @@ def find_falut_business_nic(self):
-            ip = iproute.query_links(new_nic)
-            if ip[0].state == 'DOWN':
-                self.falut_nic_count[new_nic] += 1
-            else:
-                self.falut_nic_count[new_nic] = 0
+            try:
+                ip = iproute.query_links(new_nic)
+                if ip[0].state == 'DOWN':
+                    self.falut_nic_count[new_nic] += 1
+                else:
+                    self.falut_nic_count[new_nic] = 0
+            except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)